packages/typescript-vfs/src/index.ts

```diff
@@ -91,9 +91,13 @@ export function createVirtualTypeScriptEnvironment(
   }
 }
 
+// TODO: This could be replaced by grabbing: https://github.com/microsoft/TypeScript/blob/main/src/lib/libs.json
+// and then using that to generate the list of files from the server, but it is not included in the npm package
```

Comment:
The same info is also inside the TS library itself, exported though various places (though, most internal). Perhaps that would be better for this use case?

Reply:
Yeah, if it's in the typescript compiler API in a pretty stable place we can grab it instead - you need a reference to the compiler to boot up the vfs on all platforms

Reply:
It's `@internal`, but the easiest is `ts.libMap`, which is a `Map<string, string>` from lib name to the filename. I'm also currently using it in a WIP TS monaco refactor (to load TS from `unpkg` at runtime rather than building monaco every time). Probably that's worth exporting in some form...

Reply:
I sent https://github.com/microsoft/TypeScript/pull/54011 for discussion; this variable should be present in old versions of TS (though I have not tested how far back).